core/src/main/java/com/linecorp/armeria/server/OptOutFeature.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.server;
+
+import com.linecorp.armeria.server.logging.AccessLogWriter;
+import com.linecorp.armeria.server.logging.LoggingService;
+import com.linecorp.armeria.server.metric.MetricCollectingService;
+
+/**
+ * The opt-out features that are disabled for a {@link TransientService}.
+ */
+public enum OptOutFeature {
+
+    /**
+     * Whether graceful shutdown counts the requests to the {@link TransientService} as processing requests.
```

Comment:
For all enum values: The Javadoc needs an update. For example:

    Prevents a {@link Server} from counting the requests to ...